test/TestDependenciesInMeta.py

```diff
@@ -1,35 +1,31 @@
-import os
-import unittest
+from ansiblelint import Runner
 
-from ansiblelint import Runner, RulesCollection
 
+def test_bitbucket_in_meta_dependency_is_ok(default_rules_collection):
+    filename = 'test/dependency-in-meta/bitbucket.yml'
+    good_runner = Runner(default_rules_collection, filename, [], [], [])
+    assert [] == good_runner.run()
 
-class TestDependenciesInMeta(unittest.TestCase):
 
-    def setUp(self):
-        rulesdir = os.path.join('lib', 'ansiblelint', 'rules')
-        self.rules = RulesCollection([rulesdir])
+def test_galaxy_dependency_is_ok(default_rules_collection):
+    filename = 'test/dependency-in-meta/galaxy.yml'
+    good_runner = Runner(default_rules_collection, filename, [], [], [])
+    assert [] == good_runner.run()
 
-    def test_bitbucket_in_meta_dependency_is_ok(self):
-        filename = 'test/dependency-in-meta/bitbucket.yml'
-        self.assertEqual([], Runner(self.rules, filename, [], [], []).run())
 
-    def test_galaxy_dependency_is_ok(self):
-        filename = 'test/dependency-in-meta/galaxy.yml'
-        good_runner = Runner(self.rules, filename, [], [], [])
-        self.assertEqual([], good_runner.run())
+def test_github_dependency_is_ok(default_rules_collection):
+    filename = 'test/dependency-in-meta/github.yml'
+    good_runner = Runner(default_rules_collection, filename, [], [], [])
+    assert [] == good_runner.run()
 
-    def test_github_dependency_is_ok(self):
-        filename = 'test/dependency-in-meta/github.yml'
-        good_runner = Runner(self.rules, filename, [], [], [])
-        self.assertEqual([], good_runner.run())
 
-    def test_webserver_dependency_is_ok(self):
-        filename = 'test/dependency-in-meta/webserver.yml'
-        good_runner = Runner(self.rules, filename, [], [], [])
-        self.assertEqual([], good_runner.run())
+def test_webserver_dependency_is_ok(default_rules_collection):
+    filename = 'test/dependency-in-meta/webserver.yml'
+    good_runner = Runner(default_rules_collection, filename, [], [], [])
+    assert [] == good_runner.run()
 
-    def test_gitlab_dependency_is_ok(self):
-        filename = 'test/dependency-in-meta/gitlab.yml'
-        good_runner = Runner(self.rules, filename, [], [], [])
-        self.assertEqual([], good_runner.run())
+
+def test_gitlab_dependency_is_ok(default_rules_collection):
+    filename = 'test/dependency-in-meta/gitlab.yml'
+    good_runner = Runner(default_rules_collection, filename, [], [], [])
+    assert [] == good_runner.run()
```

Comment:
This is also a single test:
```suggestion
import pytest


@pytest.mark.parametrize(
    'filename',
    (
        'bitbucket',
        'galaxy',
        'github',
        'webserver',
        'gitlab',
    ),
)
def test_external_dependency_is_ok(default_rules_collection, filename):
    playbook_path = (
        'test/dependency-in-meta/{filename}.yml'.
        format_map(locals())
    )
    good_runner = Runner(default_rules_collection, playbook_path, [], [], [])

    assert [] == good_runner.run()
```